homeassistant/components/shelly/number.py

```diff
@@ -30,12 +30,12 @@
 )
 
 
-@dataclass(frozen=True)
+@dataclass(frozen=True, kw_only=True)
 class BlockNumberDescription(BlockEntityDescription, NumberEntityDescription):
     """Class to describe a BLOCK sensor."""
 
-    rest_path: str = ""
-    rest_arg: str = ""
+    rest_path: str
+    rest_arg: str
```

Comment:
I don't know why it had an initial value, but this is changing the implementation and there are failing tests